.github/workflows/main.yml

```diff
@@ -0,0 +1,55 @@
+name: CI/CD
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Cache Bitcoin node
+        id: cache
+        uses: actions/cache@v2
+        with:
+          path: ./bitcoin-0.20.0
```

Comment:
We decided to revert to 0.20.0 because 0.21.0 didn't automatically generate wallet correct?

Reply:
Yes that's correct